templates/pricing/devices.html

```diff
@@ -29,7 +29,7 @@ <h4>SmartStart</h4>
         <hr class="u-sv1">
         <p>Let our Linux experts help you ensure the success of your application packaging on Ubuntu.</p>
 
-        <p>Enjoy snap store services for one year for up to 1,000 devices; deployment services (excluding proxy deployments); and either the packaging of up to three  application snaps (subject to approval) or equivalent consulting hours for clients that wish to do the packaging themselves with help from our top-notch Linux experts.</p>
+        <p>Enjoy snap store services, device fees and Support for one year for up to 1,000 devices; deployment services (excluding proxy deployments); and either the packaging of up to three  application snaps (subject to approval) or equivalent consulting hours for clients that wish to do the packaging themselves with help from our top-notch Linux experts.</p>
```

Comment:
I think the capital s in support is a typo, I checked the rest of the page and it's not capitalized anywhere else. I've updated the doc 👍🏻